tools/Raven.Debug/CommandLineApp.cs

```diff
@@ -139,10 +144,20 @@ private static int ExitWithError(string errMsg, CommandLineApplication cmd)
 
             var threadInfoList = new List<ThreadInfo>();
 
-            using (var dataTarget = DataTarget.AttachToProcess(processId, attachTimeout))
+            using (var dataTarget = DataTarget.AttachToProcess(processId, attachTimeout, AttachFlag.Passive))
```

Comment:
Do we want always to attach passively? IIRC it is less accurate, on Windows we could stay with what we had?

Reply:
The speed we are seeing in generating stack traces is _huge_, even if it isn't as accurate, that is far more useful